acceptancetests/assess_persistent_storage.py

```diff
@@ -165,13 +281,14 @@ def assess_persistent_storage(client):
     # Based on the test spec, persistent storage need to be tested on both
     # LXD and AWS and ebs volume is unavailable on LXD, a switcher is
     # required here to decide which test should be run.
-    environment = os.getenv('ENV', default='parallel-lxd')
```

Comment:
You are assuming a special environment that means this will fail for others who run this. Is this limited to lxd and aws only? If so, it would be nice to find a better way (check cloud perhaps, not env). I would also expect you to handle the case when it's run against something like gce (you can return an exception, or log a warning that it's not supported)